tox.ini

```
@@ -1,8 +1,11 @@
 [tox]
-envlist = py36,py37,flake8
+envlist = py{36,37}-courier_{released,master},flake8
```

Could `py{36,37}-courier_released,flake8}` be the default envlist, and `*-courier_master` be available only through `tox -e`? All this in order to lower test execution time while developing and calling `tox` locally. Travis will run all the flavors in the end.

---

Yes, although - I'm not sure how to accomplish that in a `tox.ini` config.

---

```suggestion
envlist = py{36,37}-courier_released,flake8
```

Will do the job, but `README.md` will need an update to document that `tox -e 'py{36,37}-courier_master'` can be used to test with the latest version of Operator Courier. This is required, b/c `tox -l` and `tox -a` will not list these environments.

---

Ack - should be fixed.